Monacs.Core/Option.cs

```diff
@@ -1,21 +1,40 @@
-﻿using System;
+using System;
 using System.Collections.Generic;
-using System.Linq;
 
 namespace Monacs.Core
 {
+    /// <summary>
+    /// Encapsulates optional value. A value either contains a value of a type <see cref="T"/> or it is default of <see cref="T"/>.
+    /// <para />An Option becomes Some if any value is provided. Otherwise it's None.
+    /// </summary>
+    /// <typeparam name="T">Type of optional value.</typeparam>
     public struct Option<T> : IEquatable<Option<T>>, IEquatable<T>
     {
         internal Option(T value)
         {
             Value = value;
             IsSome = true;
         }
-
+        
+        /// <summary>
+        /// Encapsulated valid value.
+        /// <para /> !!! It is not recommended to use it directly !!!

```

Comment:
Avoid using exclamation marks, we don't have to scream at the consumers ;)

Reply:
@bartsokol we cannot bold or highlight and I wanted it to be never left unnoted:)
Do you think it should be formatted as normal text?